spec/unit/provider/zone/solaris_spec.rb

```diff
@@ -149,6 +149,25 @@
         })
       end
     end
+    describe "with an invalid or unrecognized config" do
```

Comment:
Sure. An integration test could also cover the actual debug message by looking at report.logs.